test/integration/testsForApiVersion2.spec.js

```diff
@@ -0,0 +1,95 @@
+const { setupForApiVersion2 } = require('./util')
+const evrythng = require('../../dist/evrythng.node')
+
+describe('evrythng.js for apiVersion = 2', function () {
+  let settings = evrythng.setup({ apiVersion: 2, geolocation: false })
+  // await setupForApiVersion2(settings.apiUrl)
```

Comment:
Remove this commented line if not required

Reply:
fixed, thanks